projects/dashboards/ebv-analyzer/src/app/ebv-selector/ebv-selector.component.ts

```diff
@@ -194,12 +198,23 @@ export class EbvSelectorComponent implements OnInit, OnDestroy {
                                 },
                             } as MeanRasterPixelValuesOverTimeDict,
                         }),
+                        of(rasterWorkflowMetaData),
                         of(selectedCountry),
                         of(sessionToken),
                     ]),
                 ),
                 first(),
-                mergeMap(([workflowId, selectedCountry, sessionToken]) => {
+                mergeMap(([workflowId, rasterWorkflowMetaData, selectedCountry, sessionToken]) => {
+                    let spatialResolution: [number, number] = [0.1, 0.1];
+                    if (
+                        rasterWorkflowMetaData.resolution &&
+                        rasterWorkflowMetaData.resolution.x > 0.1 &&
```

Comment:
I din't think this is an issue here but 0.1° is ~160km if the layer is in EPSG:4326. If it is in UTM then it is 0.1 m. So i think we should add some more logic here. There is no projection injection in the metadata request so the SRS may be heterogenous. We could request the unit of the SRS and check if it is m or ° and than decide that...

Reply:
[e422207](https://github.com/geo-engine/geoengine-ui/pull/545/commits/e42220712b3f2ecd1cccab63a3b9a4d32e884301)